sansa-query-spark/src/main/scala/net/sansa_stack/query/spark/dof/node/Helper.scala

```diff
@@ -0,0 +1,69 @@
+package net.sansa_stack.query.spark.dof.node
+
+import java.io.{ File, PrintWriter }
+
+import org.apache.jena.graph.{ Node, Triple }
+import org.apache.spark.rdd.RDD
+
+object Helper {
+  var pw: PrintWriter = _
+  def init(outputDir: String): Unit = {
```

Comment:
These methods work well when dealing with the local file system. Usually, they can't work on the cluster (Hadoop based file system), therefore would be good to remove them and replace with the `rdd.saveAsTextFile()` instead when possible.

Reply:
removed since it is used for logging only